Functions.Templates/Templates/CosmosDBTrigger-FSharp-4.x/.template.config/template.json

```diff
@@ -0,0 +1,78 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Azure Function"],
+  "name": "CosmosDBTrigger",
+  "identity": "Azure.Function.FSharp.CosmosDBTrigger.2.x",
+  "groupIdentity": "Azure.Function.CosmosDBTrigger",
+  "shortName": "CosmosDB",
+  "tags": {
+    "language": "F#",
+    "type": "item"
+  },
+  "sourceName": "CosmosDBTriggerFSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "ConnectionStringSetting": {
```

Comment:
Same naming comments as the CSharp version of the template, ConnectionStringSetting --> Connection and Collection --> Container